apps/frontend/src/libs/components/toast-container/styles.module.css

```diff
@@ -0,0 +1,10 @@
+.toast-container {
+	--toastify-color-info: var(--color-brand-primary);
+	--toastify-color-success: var(--color-success);
+	--toastify-color-warning: var(--color-warning);
+	--toastify-color-error: var(--color-danger);
+	--toastify-toast-width: 320px;
+	--toastify-toast-min-height: 64px;
+	--toastify-toast-max-height: 800px;
```

Comment:
I'm not sure we need width / max-height set here. I think we can use the default ones

Reply:
Alright, remove them